File: problems/63/Main.py
from __future__ import print_function
import math

#--SPLIT--
#-class Solution:
    # @param a : list of integers
    # Modify the array a which is passed by reference. 
#-    def arrange(self, a):


#--SPLIT--
class Solution:
    def arrange(self, a):
        pass


#--SPLIT--
if __name__ == "__main__":
    test_cases = int(raw_input())
    for i in range(test_cases):
        lineItems = [int(l) for l in raw_input().split(" ")]
        a = lineItems[1:]
        s = Solution()
        s.arrange(a)
        for x in a:
            print("{0} ".format(x), end="")
        print("")

File: problems/63/Main.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>
#include<string.h>

//--SPLIT--

/**
 * @input a : Integer array
 * @input n1 : Integer array's ( a ) length
 * 
 * @Output Void. Just modifies the args passed by reference 
 */
//-void arrange(int* a, int n1) {
//-}


//--SPLIT--
void arrange(int* a, int n1) {
}


//--SPLIT--
int main() {
	setbuf(stdout, NULL);
	int testcases;
	scanf("%d", &testcases);
	int t = 0;
	for (t = 0; t < testcases; t++) {
		int n, i;
		scanf("%d", &n);
		int *a = (int *)malloc(sizeof(int)* n);;
		for (i = 0; i < n; i++) {
			scanf("%d", &a[i]);
		}
		arrange(a, n);
		for (i = 0; i < n; i++) {
			printf("%d ", a[i]);
		}
		printf("\n");
	}
	return 0;
}
